src/app.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const feathers = require('@feathersjs/feathers');
 const express = require('@feathersjs/express');
 
-process.env.NODE_CONFIG_DIR = path.join(__dirname, '..', 'config');
```

Comment:
isn't thjis happening automatically?
https://github.com/lorenwest/node-config/wiki/Environment-Variables#node_config_dir

Reply:
i will try it without this line

Reply:
It work with execute in cmd terminal, but not if it is start over visual studio luncher.